pkg/controller/certificates/issuing/issuing_controller.go

```diff
@@ -406,27 +403,30 @@ func (c *controller) issueCertificate(ctx context.Context, nextRevision int, crt
 // updateOrApplyStatus will update the controller status. If the
 // ServerSideApply feature is enabled, the managed fields will instead get
 // applied using the relevant Patch API call.
-func (c *controller) updateOrApplyStatus(ctx context.Context, crt *cmapi.Certificate) error {
+// conditionRemove should be true if the Issuing condition has been removed by
+// this controller. If the ServerSideApply feature is enabled and condition
+// have been removed, the Issuing condition will be set to False before
+// applying.
+func (c *controller) updateOrApplyStatus(ctx context.Context, crt *cmapi.Certificate, conditionRemoved bool) error {
 	if utilfeature.DefaultFeatureGate.Enabled(feature.ServerSideApply) {
-		crt := &cmapi.Certificate{
-			TypeMeta:   metav1.TypeMeta{Kind: cmapi.CertificateKind, APIVersion: cmapi.SchemeGroupVersion.Identifier()},
+		if conditionRemoved {
+			message := "The certificate has been successfully issued"
+			apiutil.SetCertificateCondition(crt, crt.Generation, cmapi.CertificateConditionIssuing, cmmeta.ConditionFalse, "Issued", message)
+		}
```

Comment:
I guess this boolean can be removed once the server-side apply is no longer behind feature gate.
If that is the case, might make sense to add a comment with a note/TODO about this?